docs/examples/tensorflow/demo/nvutils/runner.py

```diff
@@ -259,15 +260,23 @@ def train(infer_func, params):
         deterministic=deterministic, num_threads=num_preproc_threads,
         dali_cpu=dali_cpu, idx_filenames=train_idx_filenames)
 
+    classifier_eval, eval_input_func, eval_steps = create_validaiton_estimator(infer_func, params)
+
     try:
-        classifier.train(
-            input_fn=input_func,
-            max_steps=nstep,
-            hooks=training_hooks)
+        for i in range(num_epochs):
+            classifier.train(
+                input_fn=input_func,
+                max_steps=nstep//num_epochs,
+                hooks=training_hooks)
+            eval_result = classifier_eval.evaluate(
+                input_fn=eval_input_func,
+                steps=eval_steps)
+            print('Top-1 accuracy:', eval_result['top1_accuracy']*100, '%')
```

Comment:
Our L3 test greps for this lines, please update L3 test or make this log a bit different from the final result.